packages/extension-chakra-storefront/jest-setup.js

```diff
@@ -144,6 +144,10 @@ jest.mock('@salesforce/cc-datacloud-typescript', () => ({
 global.TextEncoder = require('util').TextEncoder
 global.TextDecoder = require('util').TextDecoder
 
+// JSDOM environment doesn't natively implement this modern Web API function.
+// Chakra v3 is using it, so we need to mock it here
+global.structuredClone = (v) => JSON.parse(JSON.stringify(v))
```

Comment:
This is actually pretty funny, the way JS doesn't have a native "deep clone" so you just chain JSON.parse and JSON.stringify